src/modules/image/index.ts

```diff
@@ -183,10 +190,10 @@ export class ImageModule {
    * Generates a random image url provided via https://picsum.photos.
    *
    * @param options Options for generating a URL for an image.
-   * @param options.width The width of the image. Defaults to `640`.
-   * @param options.height The height of the image. Defaults to `480`.
-   * @param options.grayscale Whether the image should be grayscale. Defaults to `false`.
-   * @param options.blur Whether the image should be blurred. Defaults to `false`.
+   * @param options.width The width of the image. Defaults to random integer between `1` and `3999`.
+   * @param options.height The height of the image. Defaults to random integer between `1` and `3999`.
+   * @param options.grayscale Whether the image should be grayscale. Defaults to a random boolean value.
+   * @param options.blur Whether the image should be blurred. Defaults to a random integer from `0` to `10`. `0` disables the blur.
```

Comment:
One last thing, we usually put the default sentence last.
```suggestion
   * @param options.blur Whether the image should be blurred. `0` disables the blur. Defaults to a random integer from `0` to `10`.
```

Reply:
Oh! got it